atomicapp/nulecule/container.py

```diff
@@ -160,7 +160,7 @@ def extract(self, image, source, dest, update=False):
         Utils.rm_dir(tmpdir)
 
         # Clean up dummy container
-        rm_cmd = [self.docker_cli, 'rm', '-f', container_id]
+        rm_cmd = [self.docker_cli, 'rm', '-f', container_id[:12]]
```

Comment:
> In regards to this command, is it possible to have it output via the "logger.info" command?

Actually I think that would probably not be what we want.. We want to hide some of the complexity here right? I'm about to make another comment in the generic comments that should clear up what we want to do with this PR. Please read that and give me your input.